packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -254,6 +254,10 @@ fi
 %endif
 
 %changelog
+* Thu Mar 28 2019 Shaobo Han <obo@amazon.com> - 1.27.0-1
+- Cache Agent version 1.27.0
+- Add secret support for log drivers
```

Comment:
I think this is agent feature not ecs init feature, so it should not be in ecs init log?

Reply:
Updated